posts/Fabio/2018-02-13-hibernate-ogm-5-3-CR1-released.adoc

```diff
@@ -0,0 +1,81 @@
+= "Hibernate OGM 5.3 CR1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3 CR1]
+
+== Components upgrade
+
+* Hibernate ORM version is upgraded to 5.2
+* Hibernate Search version is upgraded to 5.9
+
+== Infinispan embedded sequences and id generation improvements
+
+The Infinispan Clustered Counters, used for sequence generation, are now created at startup.
+The dialect no longer needs to check if the counter already exists for every use.
+
+When these are used for id generation their names now also contain the table name and not only the column name.
+Given the following example:
+
+====
+[source,java]
+----
+@Entity
+@Table(name = "Ruby")
+private static class Ruby {
+	@Id
+	@GeneratedValue(strategy = GenerationType.TABLE, generator = "gen2")
+	@TableGenerator(name = "gen2", table = "Ruby_generators", initialValue = INITIAL_VALUE, pkColumnValue = SEQUENCE_NAME)
+	Long id;
+}
+
+@Entity
+@Table(name = "Pearl")
+private static class Pearl {
+	@Id
+	@GeneratedValue(strategy = GenerationType.TABLE, generator = "gen3")
+	@TableGenerator(name = "gen3", table = "Pearl_generator", initialValue = INITIAL_VALUE, pkColumnValue = SEQUENCE_NAME)
+	Long id;
+}
+----
+====
+
+The generators are stored in different tables, even if they have the same pkColumnName.
```

Comment:
I would change this paragraph, with something like:

The name of the Clustered Counter will be `Ruby_generators.ruby_sequence` instead of only `ruby_sequence`.
This is to avoid collision when different @TAbleGenerator are defined with different table attribute but the same pkColumn values.

[NOTE]
Keep in mind that if you already have some generators Hibernate OGM won't recognize them. Make sure to update the model with a different initialValue higher than the one you have currently reached.